language/dsl/src/components.tsx

```diff
@@ -86,6 +87,24 @@ export const Asset = React.forwardRef<ObjectNode, AssetProps>((props, ref) => {
   const localRef = React.useRef<ObjectNode>(null);
   const Wrapper = slotContext?.wrapInAsset ? AssetWrapper : React.Fragment;
 
+  // Check to see if the applicability type is a binding using type BindingTemplateInstance
+  const isBinding = (obj: any): obj is BindingTemplateInstance => {
```

Comment:
Can we move this to a reusable utility function? There are other cases where we'll probably want to do something similar.